updater/package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
since the updater is refactored, should we change the info in here? I would say yes